main.py

```diff
@@ -88,23 +89,23 @@ def main():
 
                     logger.debug(f'Data for sell: {coin=} | {stored_price=} | {coin_tp=} | {coin_sl=} | {volume=} | {symbol=} ')
 
-                    logger.info(f'get_last_price existing coin: {coin}')
-                    last_price = get_last_price(symbol, pairing)
+                    logger.info(f'get_coin_info existing coin: {coin}')
+                    coin_info = get_coin_info(symbol, pairing)
```

Comment:
Change from a single unit of information to all the info you can get on the coin.  This is a majority of the changes, along with calling the info differently.